tests/ui/consts/const-eval/const_fn_target_feature.rs

```diff
@@ -9,7 +10,8 @@ const A: () = unsafe { ssse3_fn() };
 
 // error (avx2 not enabled at compile time)
 const B: () = unsafe { avx2_fn() };
-//~^ ERROR evaluation of constant value failed
+// FIXME: currently we do not detect this UB, since we don't want the result of const-eval
+// to depend on `tcx.sess` which can differ between crates in a crate graph.
```

Comment:
How could we fix this? Wouldn't we have to make the `const` ABI the same across the entire crate graph? And since we don't have a global coordination, wouldn't we have to ban everything except the minimum target ABI?

Reply:
> and very unlikely to ever be const.

Are we sure libs/libs-api knows that we expect that?

Reply:
I don't know... at least I don't recall anyone suggesting we make the const.